pkg/controller/intrusiondetection/intrusiondetection_controller.go

```diff
@@ -231,28 +215,54 @@ type ReconcileIntrusionDetection struct {
 	licenseAPIReady *utils.ReadyFlag
 	dpiAPIReady     *utils.ReadyFlag
 	tierWatchReady  *utils.ReadyFlag
-	usePSP          bool
+	multiTenant     bool
 	elasticExternal bool
 }
 
+func getIntrusionDetection(ctx context.Context, cli client.Client, mt bool, ns string) (*operatorv1.IntrusionDetection, error) {
+	key := client.ObjectKey{Name: "tigera-secure"}
+	if mt {
+		key.Namespace = ns
```

Comment:
We also need to make some changes to make the CRD Namespaced instead of Global for crd helm chart.